test/lib/brave.js

```diff
@@ -1073,7 +1073,7 @@ var exports = {
           logVerbose('waitForDataFile("' + dataFile + '") => ' + JSON.stringify(val.value[dataFile]))
           return val.value[dataFile] && val.value[dataFile].etag && val.value[dataFile].etag.length > 0
         })
-      }, 10000)
+      }, 30000)
```

Comment:
getting data files from the store is very slow and often leads to intermittent so I added some more time. It's often not that long

Reply:
Can this way be applied to https://github.com/brave/browser-laptop/issues/9059 ?

Reply:
may fix too but lots of bravery panel tests are intermittent I'm trying to fix them all in #11651 which includes this too